pkg/lib/cockpit.d.ts

```diff
@@ -368,4 +368,134 @@ declare module 'cockpit' {
 
     export let localStorage: Storage;
     export let sessionStorage: Storage;
+
+    /**
+     * Options passed to the HTTP client constructor.
+     */
```

Comment:
The comments are nice, but they don't have to be formatted as docstrings -- this isn't processed for documentation, and the arguments are already documented in the guide. Can you please move to single-line comments to not make this so excessively spread out?